docs/csharp/programming-guide/concepts/object-oriented-programming.md

```diff
@@ -5,8 +5,9 @@ ms.assetid: 89574786-65ef-4335-88bc-fbacd094f183
 ---
 # Object-Oriented programming (C#)
 
-C# provides full support for object-oriented programming including encapsulation, inheritance, and polymorphism.
+C# provides full support for object-oriented programming including abstraction, encapsulation, inheritance, and polymorphism.
 
+- *Abstraction* means hiding the unnecessary details from the class consumers.
```

Comment:
cc: @BillWagner @pkulikov, you may come up with a better phrasing here.

Reply:
@Youssef1313 what about

> *Abstraction* means hiding implementation details from type consumers

At least, "class" should be replaced with "type", as structure types also can abstract details away.